content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1067,6 +1067,16 @@ Grants a Role or ClusterRole within a specific namespace. Examples:
   kubectl create rolebinding myappnamespace-myapp-view-binding --clusterrole=view --serviceaccount=myappnamespace:myapp --namespace=acme
   ```
 
+Create a role binding for a particular role or cluster role. Examples:
+
+ ```shell
+  # Create a role binding for user1, user2, and group1 using the admin cluster role
+  kubectl create rolebinding admin --clusterrole=admin --user=user1 --user=user2 --group=group1
+  
+  # Create a role binding for serviceaccount monitoring:sa-dev using the admin role
+  kubectl create rolebinding admin-binding --role=admin --serviceaccount=monitoring:sa-dev
```

Comment:
Try this:
```shell
# Create a RoleBinding for user1, user2, and group1, that is bound to an existing ClusterRole named "admin".
# The RoleBinding goes into the namespaces that kubectl is targeting in its current configuration context.
kubectl create rolebinding clusteradmin --clusterrole=admin --user=user1 --user=user2 --group=group1

# Create a RoleBinding for a ServiceAccount named dev-observability, bound an existing Role named "view"
# The ServiceAccount is in the "monitoring" namespace but the Role and RoleBinding are in the "dev" namespace
kubectl --namespace dev create rolebinding dev-observability-view --role=view --serviceaccount=monitoring:dev-observability
```

SIG Auth folks, does that example help teach the concepts involved?